parliament_challenge.py

```diff
@@ -0,0 +1,140 @@
+import requests
+import waitress
+import flask
+from flask import Flask, request, json
+
+
+def filter_speeches_dict(speeches):
+    """
+    Filtering speeches data for relevant keys.
+    Relevant keys are anforande_id, dok_datum,
+    parti, avsnittsrubrik, links and intressent_id.
+    """
+
+    speeches_list = []
+    for s in speeches:
+        speechesDict = {
+                "anforande_id": s.get("anforande_id"),
+                "dok_datum": s.get("dok_datum"),
+                "parti": s.get("parti"), 
+                "avsnittsrubrik": s.get("avsnittsrubrik"),
+                "links": [{ 
+                    "rel": "speech",
+                    "href": s.get("protokoll_url_www")
+                        }],
+                "intressent_id": s.get("intressent_id")
+                }
+        speeches_list.append(speechesDict)
+    return speeches_list
+
+
+def filter_member_dict(member):
+    """
+    Filtering member data for relevant keys.
+    Relevant keys are tilltalsnamn, valkrets,
+    bild_url_192, uppgift and intressent_id.
+    """
+
+    if member["personuppgift"] is None:
+        email = "null"
```

Comment:
Set email = None, not string "null". The json dumps will automatically convert it to null.